root/static/gulpfile.js

```diff
@@ -138,14 +150,22 @@ function runYarb(resourceName, vinyl, callback) {
   });
 
   if (!DBDefs.DEVELOPMENT_SERVER) {
+    let ignoredWarning = 'Output exceeds ' + BUNDLE_MAX_LINE_LENGTH + ' characters';
+    let nodeError = console.error;
+    console.error = function(message) {
+      if (arguments && arguments[1] !== ignoredWarning) {
+        nodeError.apply(console, arguments);
+      }
+    };
+
     bundle.transform('uglifyify', {
       // See https://github.com/substack/node-browserify#btransformtr-opts
       global: true,
 
       // Uglify options
       output: {
         comments: /@preserve|@license/,
-        max_line_len: 256
+        max_line_len: BUNDLE_MAX_LINE_LENGTH
```

Comment:
I think we should perhaps remove this setting. It was to make line numbers in stack traces more usable, but I can't think of a case where it's helped that much (judging by the warnings, it's not even respecting the limit in many cases?), and this problem would be better solved by [providing source maps to Sentry](https://docs.sentry.io/clients/javascript/sourcemaps/).

Reply:
As stated in the commit message, there is no other way to _fix the `max_line_len` warning_ except by removing the `max_line_len` option. Do we remove this option now? Or do we merge this work around until we provide source maps to Sentry?

Reply:
I'd personally remove `max_line_len` now; it's broken and not useful. Providing source maps will have to wait for Webpack, since it's too difficult with our current setup.